lib/Cake/Network/Email/CakeEmail.php

```diff
@@ -1203,7 +1203,7 @@ public function send($content = null) {
  * @throws SocketException
  */
 	public static function deliver($to = null, $subject = null, $message = null, $transportConfig = 'fast', $send = true) {
-		$class = __CLASS__;
+		$class = get_called_class();
```

Comment:
@markstory no way to disable the branch?

https://stickler-ci.com/docs#config

> branches:
    ignore: ['2.x', '2.next']

Wouldn't that work?